railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
Also, bundler uses threads instead of processes to parallelize since 1.8.0 ([changelog](https://github.com/bundler/bundler/blob/master/CHANGELOG.md#180pre-2015-01-26), https://github.com/bundler/bundler/pull/3039). So the number of cpus available on one's machine doesn't actually matter.

I would let bundler use the global config rather than changing rails.